bin/collator/src/parachain/service.rs

```diff
@@ -1285,84 +628,44 @@ where
         relay_chain_slot_duration: Duration::from_secs(6),
         proposer: cumulus_client_consensus_proposer::Proposer::new(proposer_factory),
         collator_service,
-        authoring_duration: Duration::from_millis(1500),
+        authoring_duration: Duration::from_millis(2000),
```

Comment:
that's the correct value: 2s
Previously we added 1.5s as mention on implementation guide and it is fine if collators are running powerful machines but give reality they're not, it will be better assume machine spec is minimal and 2s time frame will match 2sec of block weight.